src/index.js

```diff
@@ -0,0 +1,48 @@
+// import _ from 'lodash';
+import './style.css';
+
+const form = document.getElementById('form');
+const tasks = [
+  {
+    description: 'attending mornning session',
+    completed: true,
+    index: 0,
+
+  },
+  {
+    description: 'coding',
+    completed: true,
+    index: 1,
+
+  },
+  {
+    description: 'break dance',
+    completed: false,
+    index: 0,
+
+  },
+];
+
+// console.log(tasks)
```

Comment:
Please remove this commented line of code to keep your codebase organized and clean.